web/packages/core/src/ruffle-player.ts

```diff
@@ -516,6 +520,20 @@ export class RufflePlayer extends HTMLElement {
                 (ruffleConstructor.is_wasm_simd_used() ? "ON" : "OFF") +
                 ")"
         );
+        {
+            const actuallyUsedRenderer =
+                this.instance!.get_actually_used_renderer() as
+                    | RenderBackend
+                    | null
+                    | "";
+
+            if (actuallyUsedRenderer) {
+                console.log(
```

Comment:
I support n0samu's idea to display the used renderer in the `New Ruffle instance created` log above. I think we'd want to reduce the amount of explicit `console.log()`s to the minimum.

Reply:
It is good that you mention it again, because I somehow completely missed that it would fit well there :sweat_smile: (I think I was distracted or focused on other aspects). I will change it to be part of that.

Reply:
![new browser console logging](https://user-images.githubusercontent.com/131509408/235734998-7bb5f405-6158-42fe-8c80-9002b9f67332.png)